src/sentry/static/sentry/app/stores/groupingStore.jsx

```diff
@@ -330,14 +336,6 @@ const GroupingStore = Reflux.createStore({
     return promise;
   },
 
-  onMergeSuccess(id, ids, response) {
```

Comment:
This was never getting fired because the action would happen on `GroupStore`, not `GroupingStore`.